### PR TITLE
type issue fixed, closes #131

### LINKED ISSUE #131: types issue: Cannot find module './types/types'
```
node_modules/.pnpm/@editorjs+embed@2.7.4/node_modules/@editorjs/embed/dist/index.d.ts:1:51 - error TS2307: Cannot find module './types/types' or its corresponding type declarations.

1 import { ServiceConfig, ServicesConfigType } from './types/types';
```

### PR DESCRIPTION
**Pull request by Sozialarchiv**
This commit fixes an issue with type import:

```
node_modules/.pnpm/@editorjs+embed@2.7.4/node_modules/@editorjs/embed/dist/index.d.ts:1:51 - error TS2307: Cannot find module './types/types' or its corresponding type declarations.

1 import { ServiceConfig, ServicesConfigType } from './types/types';
```

Types should not use here as a folder name.

Resolves #131

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@editorjs/embed",
-  "version": "2.7.5",
+  "version": "2.8.0",
```
Comment (neSpecc) [inline package.json:3]:
```suggestion
  "version": "2.7.6",
```

File: src/index.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import SERVICES from './services';
 import './index.css';
 import { debounce } from 'debounce';
-import { ServiceConfig, ServicesConfigType } from './types/types';
-import { API , PatternPasteEventDetail } from '@editorjs/editorjs';
+import { type ServiceConfig, type ServicesConfigType } from './serviceConfig';
```
Comment (neSpecc) [inline src/index.ts:4]:
```suggestion
import type { ServiceConfig, ServicesConfigType } from './serviceConfig';
```

File: src/index.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+import { type API , type PatternPasteEventDetail } from '@editorjs/editorjs';
```
Comment (neSpecc) [inline src/index.ts:5]:
```suggestion
import type { API, PatternPasteEventDetail } from '@editorjs/editorjs';
```

File: src/index.ts (hunk 2 above)
Comment (Sozialarchiv) [reply src/index.ts:4]: Good one. I will do that.

**Comment by Sozialarchiv** [comment]
I added all your suggestions.

**Review by neSpecc — APPROVED** [review]

**Pull request merged**